content/kubermatic/master/architecture/_index.en.md

```diff
@@ -5,4 +5,10 @@ weight = 10
 chapter = true
 +++
 
-# Architecture
+## Architecture
+
+This section contains information about the architecture of KKP clusters, such as the hardware requirements for the master cluster.
```

Comment:
the text formatting looks strange. Like this:
------------
   -----
     --

Reply:
Thank you! Fixed it.